Update/some_003.txt

```diff
@@ -2234,16 +2234,16 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>畠山老</color>", NULL, "<color=#f5e6d3>Hatakeyama</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_hatakeg1020", 256, TRUE);
 	OutputLine(NULL, "「まぁ、都会じゃと何かと物入りで、息子の稼ぎだけでは苦しいからの。",
-			NULL, "\"Well, in this city with all its expenses, it is painful relying on only my son's earnings.", Line_WaitForInput);
+			NULL, "\"Well, in this city with all its expenses,  painful relying on only my son's earnings.", Line_WaitForInput);
```

Comment:
Sorry, messed up here. It should be "Well, in this city with all its expenses, it's hard for my son to live on his earnings alone."